src/ArrangerConduit.sol

```diff
@@ -0,0 +1,251 @@
+                continue;
+            }
+
+            uint256 fillAmount = fundRequest.amountRequested - fundRequest.amountFilled;
+
+            if (fillAmount > fundsRemaining) {
+                fillAmount = fundsRemaining;
+                fundRequest.status = StatusEnum.PARTIAL;
+            } else {
+                fundRequest.status = StatusEnum.COMPLETED;
+                newStartingFundRequestId++;
+            }
+
+            fundsRemaining           -= fillAmount;
+            fundRequest.amountFilled += fillAmount;
+            totalWithdrawable[asset] += fillAmount;
+
+            maxWithdraw[fundRequest.ilk][asset] += fillAmount;
+
+            if (fundsRemaining == 0) break;
+        }
+
+        startingFundRequestId[asset] = newStartingFundRequestId;
+
+        require(
+            ERC20Like(asset).transferFrom(fundManager, address(this), returnAmount),
+            "Conduit/transfer-failed"
+        );
+    }
+
+    function payInterest(address asset, uint256 amount) external override isFundManager {
+        totalInterestEarned[asset] += amount;
+
+        require(
+            ERC20Like(asset).transferFrom(fundManager, admin, amount),
+            "Conduit/transfer-failed"
+        );
+    }
+
+    /**********************************************************************************************/
+    /*** View Functions                                                                         ***/
+    /**********************************************************************************************/
+
+    function maxDeposit(bytes32 ilk, address asset)
+        external override pure returns (uint256 maxDeposit_)
+    {
+        ilk; asset;  // Silence warnings
+        maxDeposit_ = type(uint256).max;
+    }
+
+    function isCancelable(address asset, uint256 fundRequestId)
+        external override view returns (bool isCancelable_)
+    {
+        StatusEnum status = fundRequests[asset][fundRequestId].status;
+
+        isCancelable_ = status == StatusEnum.PENDING || status == StatusEnum.PARTIAL;
+    }
+
+    function activeFundRequests(address asset, bytes32 ilk)
+        external override view returns (
+            uint256[] memory fundRequestIds,
+            uint256 totalRequested,
+            uint256 totalFilled
+        )
+    {
+        uint256 i;
+
+        for (uint256 j = startingFundRequestId[asset]; j < fundRequests[asset].length; j++) {
+            FundRequest memory fundRequest = fundRequests[asset][j];
+
+            if (
+                (
+                    fundRequest.status == StatusEnum.PENDING ||
+                    fundRequest.status == StatusEnum.PARTIAL
+                ) &&
+                fundRequest.ilk == ilk
+            ) {
+                fundRequestIds[i++] = j;
+
+                totalRequested += fundRequest.amountRequested;
+                totalFilled    += fundRequest.amountFilled;
+            }
+        }
+    }
+
+    function totalActiveFundRequests(address asset)
```

Comment:
I think we need to adjust these views to provide two important numbers to the fund manager:

`pendingDeposits(asset)` - the amount the fund manager should deposit into the off-chain treasury investment.
`pendingWithdrawals(asset)` - the amount the fund manager should withdraw and return to the pool at the next available opportunity.